llmfoundry/callbacks/model_gauntlet_callback.py

```diff
@@ -27,7 +26,7 @@ class ModelGauntlet(Callback):
     specification provided in the constructor.
 
     Args:
-        logger_keys (dict): These are the exact keys that the individual benchmark metrics will be
```

Comment:
This is API breaking -- is this an absolutely neccessary change, and if so, can we have backwards compatibility with a deprecation warning?

Reply:
It's not strictly necessary, so I can change it back!